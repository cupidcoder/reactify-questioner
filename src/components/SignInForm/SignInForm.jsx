import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import Button from '../Button/Button';
import style from './SignInForm.css';

class SignInForm extends Component {
  render() {
    return (
      <div className={style.SignInFormContainer}>
        <div className={style.SignInForm}>
          <h2>Sign In</h2>
          <form>
            <div className={style.Input}>
              <input type='email' id='email' placeholder='Email address' />
            </div>
            <div className={style.Input}>
              <input type='password' id='password' placeholder='Password' />
            </div>
            <div className={style.Button}>
              <Button>Sign in</Button>
            </div>
            <p className={style.FooterText}>
              Don&rsquo;t have an account? Sign up
              {' '}
              <Link to='/signup'>here</Link>
            </p>
          </form>
        </div>
      </div>
    );
  }
}

export default SignInForm;
