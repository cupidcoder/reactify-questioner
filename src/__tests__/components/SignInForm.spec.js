import React from 'react';
import { shallow } from 'enzyme';
import SignInForm from '../../components/SignInForm/SignInForm';

describe('SignInForm', () => {
  it('Should correctly save snapshot of the SignInForm', () => {
    const wrapper = shallow(<SignInForm />);
    expect(wrapper).toMatchSnapshot();
  });
});
