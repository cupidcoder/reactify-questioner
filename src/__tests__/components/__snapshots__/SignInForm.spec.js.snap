// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignInForm Should correctly save snapshot of the SignInForm 1`] = `ShallowWrapper {}`;
